src/schema/rules/nifti_header.yaml

```diff
@@ -0,0 +1,40 @@
+---
+nifti_header:
```

Comment:
Should these all be defined in the objects folder instead? Perhaps we need a new `schema/objects/data_file_headers.yaml` or something.